Tools/readvar.py

```diff
@@ -59,7 +66,7 @@ def readvar(varlist, config, logfile):
             if "land" in f[varname_clim[index]].dimensions:
                 land = f["land"][:] - 1
                 ntime = len(da)
-                uncomp = np.ma.masked_all((ntime, nlat * nlon))
+                uncomp = np.arr.masked_all((ntime, nlat * nlon))
```

Comment:
Does this line make sense? I haven't heard of `np.arr` before.

Reply:
@tztsai bump

Reply:
I think it does not make sense